services/validation/README.md

```diff
@@ -1,15 +1,128 @@
-# Sourcify Script 🧑‍💻📝🔍 
+# Sourcify Validation 🧑‍💻📝🔍 
 
-#### Use this script to verify if your source files are in the correct format for [Sourcify](https://github.com/ethereum/sourcify)
+The Sourcify Validation script helps you re-compile and source-verify your Solidity smart contracts.
 
-##### How to?
+If Solidity source files are spread across multiple files, re-compiling them is often complicated.
+To help with re-compilation, [the Solidity compiler](https://solidity.readthedocs.io/en/latest/installing-solidity.html) generates a file called "metadata.json" that
+contains all settings, filenames and file hashes required to re-compile the contract into the exact
+same binary.
+
+Several tools, including [Truffle](https://www.trufflesuite.com/) and [Buidler](https://buidler.dev) store this file with each compilation.
+
+## Usage as a script
 
 * `npm install -g sourcify-validation`
-* `sourcify-validation path/to/file1 path/to/file2 path/to/dir path/to/zip`
-    * the script will scan through all the provided files, directories and zips searching for metadata and the belonging source files
-* If everything goes well, proceed here: [Sourcify](https://verification.komputing.org/)
+* `sourcify-validation my/repository`
+
+You can specify a sequence of files, directories and even zip files.
+The script will scan through all the provided files, searching for metadata and the belonging source files.
+
+If everything goes well, you can ask it to provide a "standard-json input file"
+that you just need to send to the compiler, etherscan, or whatever service you use
+for verification.
+
+Verification is also achievable through the use of [Sourcify](https://verification.komputing.org/). All you need to do is upload the files
```

Comment:
Shouldn't we use `sourcify.dev`?

Reply:
That makes sense; this old link was present here even before the documentation was handed over to me.